src/leds-hybris.cpp

```diff
@@ -97,7 +97,13 @@ void LedsHybris::turnOn()
     // pulse
     light_state_t state;
     memset(&state, 0, sizeof(light_state_t));
-    state.color = m_color;
+    int alpha = (m_color >> 24) & 0xFF;
+    int red = (m_color >> 16) & 0xFF;
+    int green = (m_color >> 8) & 0xFF;
+    int blue = m_color & 0xFF;
+    alpha = 0xff; //TODO: Support alpha
+    int newcolor = ((alpha & 0xff) << 24) + ((red & 0xff) << 16) + ((green & 0xff) << 8) + (blue & 0xff);
+    state.color = newcolor;
```

Comment:
Please use the m_colors built in functions see https://github.com/ubports/hfd-service/blob/xenial/src/rgba.h

But this is easier to do by just adding the alpha channel, something like this

```suggestion
    state.color = ((0xff & 0xff) << 24) | m_color;
```